pkg/crd/spec.go

```diff
@@ -151,14 +160,18 @@ func (p *Parser) NeedCRDFor(groupKind schema.GroupKind, maxDescLen *int) {
 	hasStorage := false
 	for _, ver := range crd.Spec.Versions {
 		if ver.Storage {
+			// more than one storage, already set it true on previous iteration
+			if hasStorage {
+				errs = append(errs, fmt.Errorf("CRD for %s has more than one storage version", groupKind))
```

Comment:
sets.New[String] should do it?

Reply:
I think there is a misunderstanding here.

when you loop through different version, it would be something like `v1`, `v2`, `v3`, and if we put them all in the set, it would be something like `set{"v1,"v2", "v3"}`. **Storage Version** is a boolean value, making it a map would be `map{"v1": true,"v2": true, "v3": false}`, I don't think there is need to set or map. We would need to check if the boolean is triggered twice, that should be more than enough